tracking/js/third-party/mouseflow.js

```diff
@@ -6,14 +6,45 @@ const getCookieValue = function (key) {
 	return (a) ? a[1] : undefined;
 }
 
-// Loads session cam tracking code
+function enableMouseflow () {
+	window._mfq = window._mfq || [];
+	window._mfq.push(['setVariable', 'spoorId', getCookieValue('spoor-id')]);
+	loadScript('https://cdn.mouseflow.com/projects/3d6fc486-2914-4efc-a5ae-35a5eac972f2.js');
+}
+
+// Loads mouseflow tracking code
 module.exports = function (flags) {
 
-	if (flags && (flags.get('mouseflowForce') || flags.get('mouseflow'))) {
+	const isSignUpApp = !!document.querySelector('html[data-next-app=signup]');
 
-		window._mfq = window._mfq || [];
-		window._mfq.push(['setVariable', 'spoorId', getCookieValue('spoor-id')]);
-		loadScript('https://cdn.mouseflow.com/projects/3d6fc486-2914-4efc-a5ae-35a5eac972f2.js');
+	if (flags.get('mouseflowForce')) {
+		enableMouseflow();
+	}
+	else if (flags.get('mouseflow')) {
 
+		if (isSignUpApp) {
+			enableMouseflow();
+		}
+		else {
+			fetch('https://session-next.ft.com/', {
+				timeout: 2000,
+				credentials: 'include'
+			})
+			.then (function (response) {
+				switch (response.status) {
+					case 404:
+						return {};
+					case 200:
+						return response.json();
+					default:
+						throw new Error(`${response.status} - ${response.statusTest}`);
+				}
+			})
+			.then (function (session) {
+				if (parseInt(session.passportId) % 100 === 0) { // 1%
```

Comment:
@commuterjoy how is uuid generated? Can I safely `mod 100` over the decimal value of the last segment, ie `86976528967924` for `4f1acd1ce8f4` of `c58d7528-d4d5-4cd3-aff8-4f1acd1ce8f4`?